test/invalid-store.test.ts

```diff
@@ -10,17 +11,19 @@ afterEach(() => {
 it('should reset the store if contains an invalid value', () => {
   jest.spyOn(global.console, 'warn').mockImplementation(() => {})
   localStorage.setItem(PERSISTENCY_KEY, 'not valid json')
-
+  const mockConsole = mocked(global.console.warn)
   expect(() => store.hydrate()).not.toThrow()
   expect(localStorage.getItem(PERSISTENCY_KEY)).toBeNull()
-  expect(console.warn).toHaveBeenCalledWith(`
+  const errorMessage = mockConsole.mock.calls[0][0]
+  expect(errorMessage).toContain(`
 [virtual-cookie] Failed to parse a stored cookie from the localStorage (key \"MSW_COOKIE_STORE\").
 
 Stored value:
-not valid json
-
-Thrown exception:
-SyntaxError: Unexpected token o in JSON at position 1
-
-Invalid value has been removed from localStorage to prevent subsequent failed parsing attempts.`)
+not valid json`)
+  // not asserting on the entire error message because it's different in different node versions
+  expect(errorMessage).toContain(`Thrown exception:
+SyntaxError: Unexpected token`)
+  expect(errorMessage).toContain(
+    `Invalid value has been removed from localStorage to prevent subsequent failed parsing attempts.`,
+  )
```

Comment:
in node 20 it seems the formatting of the `o in JSON at position` is slightly different.  I don't know that we need to assert the whole mesage, so I was a bit broader

Reply:
We can use `.toContain`, just like you did above! 👌